stdpopsim/catalog/BosTau/demographic_models.py

```diff
@@ -105,3 +105,550 @@ def _HolsteinFriesian_1M13():
 
 
 _species.add_demographic_model(_HolsteinFriesian_1M13())
+
+
+def _HolsteinFriesian_1B16():
+    id = "HolsteinFriesian_1B16"
+    description = (
+        "Piecewise size model for Holstein-Friesian cattle (Boitard et al., 2016)."
+    )
+    long_description = """
+    The piecewise-constant population size model of Holstein-Friesian cattle
+    from Boitard et al. (2016). Effective population sizes were estimated using
+    Approximate Bayesian Computation approach with allele frequency spectrum and
+    linkage-disequlibrium statistics (both on SNPs with minor allele frequency
+    above 0.2) observed in 25 individuals,
+    using the following assumptions:
+    a generation interval of 5 years (page 10),
+    a mutation rate of 1e-8 (pages 8 and 22), and
+    a recombination rate of 3.66e-9 (page 15).
+    Effective population sizes are given in Figure 6,
+    with exact values provided by the lead author in personal communication.
+    The 25 individuals' genomes were obtained from the 1000 bull genomes Run 2
+    (of which 129 were Holstein-Friesian;
+    the authors chose 52 from Australia to get a homogeneous sub-population,
+    kept the least inbred and unrelated samples, and from these randomly selected 25).
+    """
+    populations = [
+        stdpopsim.Population(id="Holstein_Friesian", description="Holstein-Friesian"),
+    ]
+    citations = [
+        stdpopsim.Citation(
+            author="Boitard et al.",
+            year=2016,
+            doi="https://doi.org/10.1371/journal.pgen.1005877",
+            reasons={stdpopsim.CiteReason.DEM_MODEL},
+        )
+    ]
+    return stdpopsim.DemographicModel(
+        id=id,
+        description=description,
+        long_description=long_description,
+        populations=populations,
+        citations=citations,
+        generation_time=_species.generation_time,
```

Comment:
This is said above? Even if this agrees with the species definition, if it's specified in the model it should probably be put here? (However, does the generation time actually affect the model inference, as times are in generations? If not, maybe remove it from the parameter table above.)
```suggestion
        generation_time=5,
```

Reply:
Will do